### ISSUE TITLE
Make it More Apparent that SECRET_KEY cannot be blank

### ISSUE THREAD

**Issue by ZaxLofful** [voxpupuli/puppetboard#978]
The main README doesn't say anything about how the value of SECRET_KEY cannot be blank. Nor does the documentation for Puppet show in the example that the key is needed to run.

When we attempt to run this container, it fails unless we add an arbitrary SECRET_KEY of any kind, but the documents don't say that.

Was able to figure it out by going through the code and seeing a comment that says:

# this empty string has to be changed, we validate it with check_secret_key()

This should be on the main page in bold letters, because it just doesn't work otherwsie.

**Comment by kenyon** [comment]
The README does say something about it, but it doesn't say it can't be the empty string: https://github.com/voxpupuli/puppetboard/blob/master/README.md?plain=1#L171-L173

Also, if you use the [puppetboard module](https://forge.puppet.com/modules/puppet/puppetboard/readme), this is handled for you.

**Comment by ZaxLofful** [comment]
Yeah, that's what I was implying. On the README make a part about that it cannot be blank. It took me DEV team about a week to figure out why it wasn't launching because its not clear from standard run guide.

Will look into the other solution as well.

**Comment by kenyon** [comment]
See #978 for an update to the README.
